<launch>
  <group>
    <node name="image_proc" pkg="image_proc" type="image_proc" ns="usb_cam">
      <remap from="image_raw" to="/usb_cam/image_raw" />
      <remap from="camera_info" to="/usb_cam/camera_info" />
      </node>
    <!--
    <node name="image_view" pkg="image_view" type="image_view" >
      <param name="autosize" value="true" />
      <remap from="image" to="/usb_cam/image_rect_color"/>
      </node>
    -->
  </group>
</launch>
